test/integration.test.js

```diff
@@ -426,9 +444,10 @@ test("Comment and add labels on PR included in the releases", async (t) => {
       {
         permissions: { push: true },
         full_name: `${owner}/${repo}`,
+        git_url: `htttps://api.github.local/${owner}/${repo}.git`,
       },
       {
-        // TODO: why do we call the same endpoint twice?
+        // TODO: why do we call the same endpoint three times?
```

Comment:
```suggestion
```

Please remove the comment at this point 😉, the question is already answered haha 😃

Reply:
well, `GET /repos/${owner}/${repo}` still gets called twice in many tests for some other reason

Reply:
confirmed, `fail.js` and `success.js` do `GET /repos/{owner}/{repo}`

Reply:
anyway I did remove the comment